docs/INSTALL.md

```diff
@@ -102,36 +102,76 @@ command line scripts as explained in the [scripts README](https://github.com/Ada
     ```
 
 ### Installation on Windows
-> note: Installing JoinMarket on Windows using the following method requires Windows 10 version 1607 or later.
 
-#### Enable Windows Subsystem for Linux
+Before starting, note you need either (a) Bitcoin Core installed on Windows or (b) use a remote connection to Bitcoin Core specified in the `joinmarket.cfg` (explained at the end of this section).
+
+If (a), then note the following two points:
+
+##### Installing Bitcoin Core
+If you haven't done so yet, install Bitcoin Core as described [here](https://bitcoin.org/en/full-node#windows-10). After starting it for the first time, it will start the Initial Block Download. JoinMarket cannot be used until this is finished. More information on that can be found [here](https://bitcoin.org/en/full-node#initial-block-downloadibd).
+
+##### Configuring Bitcoin Core
+Bitcoin Core needs to be configured to allow JoinMarket to connect to it. From the `Settings` menu choose `Options` and click `Open Configuration File`. Add `server=1`, save and close the file. After that restart Bitcoin Core.
+
+There are currently two choices for installing on Windows; one, directly installing on Windows, requiring the manual addition of a libsodium dependency, or, two, using Ubuntu via the WSL mechanism (which may require additional setup to make the Qt GUI work).
+
+1) [Installation directly on Windows](#installation-directly-on-windows)
+
+2) [Installation using WSL](#installation-using-wsl)
+
+#### Installation directly on Windows
+
+First, if you have not done so, install [Python](https://www.python.org/downloads/windows/) - specifically, the latest Python 3 version. Make sure to choose to install `pip` during the installation (it should be included automatically, but you can choose the custom installation option to choose it).
+
+Be sure to choose the option that allows `python` to be in your PATH variable (you are prompted to do this at the end of the install).
+
+Install Joinmarket, choosing the zip file of the latest [release](https://github.com/JoinMarket-Org/joinmarket-clientserver/releases). You should check the .asc signature file on the zip that you download.
+
+Unzip the `joinmarket-clientserver-x.x.x` (where `x.x.x` is the release number) in any appropriate location.
+
+Using the command prompt in Administrator mode, go to that directory and run the commands:
+
+`pip install -r requirements\base.txt`
+`pip install -r requirements\gui.txt`
+
+(the latter is needed for Joinmarket-Qt).
+
+The final step is to manually add the libsodium dependency, as mentioned. Do the following:
+
+Download the file at `https://www.nuget.org/api/v2/package/libsodium` and rename it to `.zip` so that you can unzip it. Once unzipped, find the `libsodium.dll` file at `runtimes\win-x64\native\libsodium.dll` and copy it into `C:\Windows\System` (note this will require Admin rights).
```

Comment:
Haven't tested this yet, but both this and `pip install`'s above could be added to some single `install.bat` file for Windows users, not? (not 100% sure about downloading file from Windows cmd, haven't written any scripts for Windows for a long time; probably could use PowerShell instead).

Reply:
Yes, I had the same thought.